src/Photos/PHChangeRequest.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+
+namespace Photos {
+
+	public partial class PHChangeRequest {
+#if !XAMCORE_3_0
```

Comment:
why not `XAMCORE_4_0` ?

`_3_0` was used for changes that are included in tvOS and watchOS (but could not be enabled in iOS and macOS without braking changes). It should not (99% of time) be used again.

Reply:
Mmm I'm not sure, @rolfbjarne sent me that code snippet, but I can change it to be XAMCORE_4_0 if need be.

Reply:
It's `3_0` because it's required for this code, which is using `3_0`: https://github.com/xamarin/xamarin-macios/blob/846bc763a881855bc455848ef15d2c78b765ffc0/src/Photos/PHAssetChangeRequest.cs#L9-L14